# Define the goal
float32 goal_angle
float32 goal_distance
float32 angular_speed
float32 linear_speed
---
# Define the result
uint32 result_flag
float32 result_angle
float32 result_distance
---
# Define a feedback message
float32 current_angle
float32 current_distance

